app/components/pages-menu/component.js

```diff
@@ -4,6 +4,65 @@ export default Ember.Component.extend({
     style: Ember.computed('layer.settings.values.color', function(){
         return Ember.String.htmlSafe('color: ' + (this.get('layer.settings.values.color') || '#333') + '; ');
     }),
+    sticky: Ember.observer('layer.settings.values.stickToTop' , 'layers.[]' , function() {
+        let topOfNav = null;
+        if( $('.pages-menu')[0] ){
+            topOfNav = $('.pages-menu').offset().top;
+        }
+        let self = this;
+        if(this.get('layer.settings.values.stickToTop')){
+            $(window).on('scroll.nav', function (e) {
+
+
+                let _topOfNav = null;
+                _topOfNav = topOfNav;
+                if($('.editMenu')[0]){
+                    _topOfNav = topOfNav - 51;
+                    if (!self.get('editMode')) {
+                        _topOfNav = topOfNav - 120 ;
```

Comment:
Why is this different when editmode is on? Isn't the height of edit menu the same regardless?